app/controllers/entities_controller.rb

```diff
@@ -3,7 +3,7 @@ class EntitiesController < ApplicationController
   include CacheHelper
   include Europeana::EntitiesApiConsumer
 
-  attr_reader :body_cache_key
+  attr_reader :body_cache_key, :entity_caching_enabled
```

Comment:
There's no use in having an attr_reader if the method is implemented in the class.